src/solvers/refinement/string_refinement.cpp

```diff
@@ -1935,7 +1935,7 @@ static optionalt<exprt> find_counter_example(
   const exprt &axiom,
   const symbol_exprt &var)
 {
-  satcheck_no_simplifiert sat_check;
+  satcheck_no_simplifiert sat_check(0., 0.);
   bv_refinementt::infot info;
```

Comment:
How are these values related to the satcheck_minisat2_baset::default_random_seed.  Also, this is probably the one place this patch is actually needed.